pkg/gitprovider/bitbucket.go

```diff
@@ -10,7 +10,7 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/ktrysmt/go-bitbucket"
+	bitbucket "github.com/ktrysmt/go-bitbucket"
```

Comment:
No more need for this change since you moved the server provider to the other file. Please revert.

Reply:
reverted